src/views/components/admin/banUserForm.js

```diff
@@ -56,7 +56,7 @@ function BanUserForm() {
                 </Form.Group>
                 <br></br>
                 <Button variant="primary" type="submit" data-test-id='select-ban-user-submit'>
-                    Delete
```

Comment:
Is this needed now, given that we have the 'Ban' button in the drivers list component